doc/nrf/releases_and_maturity.rst

```diff
@@ -19,6 +19,13 @@ Major and minor releases always come with their respective :ref:`migration_guide
 
 If an issue is found in a release after it has taken place, those issues are listed on the :ref:`known_issues` page.
 
+* Updated:
```

Comment:
Please move this content to the changelog.

Reply:
outstanding comment